vars/Suse.yml

```diff
@@ -27,5 +27,8 @@ os_auth_sys_uid_max: 499
 os_auth_sys_gid_min: 100
 os_auth_sys_gid_max: 499
 
+# defaults for useradd
+os_useradd_create_home: false
```

Comment:
@Aisbergg why Suse in particular has this disabled by default?

Reply:
> @Aisbergg why Suse in particular has this disabled by default?

That I can't say for sure, all I can say is that those are the defaults found in the [shadow package](https://opensuse.pkgs.org/15.2/opensuse-oss-x86_64/shadow-4.6-lp152.3.62.x86_64.rpm.html).